R/fit.R

```diff
@@ -38,7 +38,7 @@ epidist <- function(data, formula, family, prior, backend, fn, ...) {
 #' @export
 epidist.default <- function(data, formula = mu ~ 1,
                             family = "lognormal", prior = NULL,
-                            backend = "cmdstanr", fn = brms::brm, ...) {
+                            backend = "rstan", fn = brms::brm, ...) {
```

Comment:
You could list out the options as `backend = c("rstan", "cmdstan")`and use `backend <- match.arg(backend)` to pick out the input. See [recommendations here](https://design.tidyverse.org/enumerate-options.html).

Reply:
This is very good, thanks James!

Reply:
Sorry, I didn't realise it was still a draft PR, hence not ready for review😬 .